src/core/hle/kernel/thread.h

```diff
@@ -568,9 +570,62 @@ class Thread final : public SynchronizationObject {
         return has_exited;
     }
 
+    struct QueueEntry {
+    private:
+        Thread* prev;
+        Thread* next;
+
+    public:
+        constexpr QueueEntry() : prev(nullptr), next(nullptr) { /* ... */
```

Comment:
```suggestion
        constexpr QueueEntry() = default;
```

Given the assignments can be moved to in-class initializers.